### DISCUSSION TITLE
Translation of validation messages

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#380] [Q&A] [answered]
Hi Labs !
I send to validation the same ( almost .. ) notice subType 1, twice, first with english locale  and the second with italian.
Same **<svrl:failed-assert id="BR-BT-00738-0053" ...role="ERROR">**
en : 
 <svrl:text>Notice Publication Date Preferred (BT-738-notice) shall be within 3 months after the Notice Dispatch Date (BT-05-notice)</svrl:text>
 it  : 
 <svrl:text>rule|text|BR-BT-00738-0053</svrl:text>
     
There are not the rules translated in italian ? Will they be translated in future ?
Thanks and Regards 
Renato

**Comment by bertrand-lorentz** [comment, marked as answer]
The messages for rules will be translated in all EU official languages, and we are currently working on this. Currently the messages are only fully available in English, with some machine translations in French and Greek, and just one single machine translation for the other languages.
The translations are available in the SDK in the files `rule_*.xml` in the `translations` folder. The validation service uses those files to provide translations, so you can keep an eye on those to see when new translations are added.
